README.md

```diff
@@ -105,12 +105,13 @@ supported on EOL Ubuntu releases so make sure you're on a recent release of Ubun
 Master and all slaves:
 
     wget -qO - https://deb.nodesource.com/setup_12.x | sudo -E bash -
-    sudo apt install -y nodejs python-dev git build-essential
-    git clone -b master https://github.com/clusterio/factorioClusterio.git
-    cd factorioClusterio
+    sudo apt install -y nodejs
+    mkdir clusterio
+    cd clusterio
+    npm init -y
+    npm install @clusterio/master @clusterio/slave @clusterio/ctl
```

Comment:
Possibly use npm init @clusterio ?

Reply:
You will have to decide if you want to add that here or later. Either makes sense.

Reply:
Can't do that until the `@clusterio/create` exists, which it doesn't.  This is a method that can be used until such a package is created.

Reply:
To reiterate, the documentation should reflect the current state of the project.  Not some future idea that may or may not have been implemented.  If the create package is implemented then the documentation here will be changed to reflect that.